src/db.ls

```diff
@@ -67,16 +67,58 @@
     db.DB = {}
     minimatch = require \minimatch
     try
-      db.DB = JSON.parse do
-        require \fs .readFileSync "#dataDir/dump.json" \utf8
-      console.log "==> Restored previous session from JSON file"
+      db.DB =
+        save_timestamps: {}
+        timestamps: {}
+      if fs.existsSync "#dataDir/dump/"
+        fs.readdirSync("#dataDir/dump/").forEach (f) ->
```

Comment:
It would be nice to filter out [dot/hidden files](https://en.wikipedia.org/wiki/Dot-file), e.g.

```ls
f <-! fs.readdir-sync "#dataDir/dump/"
        .filter (/^[^.]/ .test _)            
        .for-each
# rest of code ...
```